src/renderer/html_handlebars/hbs_renderer.rs

```diff
@@ -903,28 +903,42 @@ fn hide_lines(content: &str) -> String {
     }
 
     let mut result = String::with_capacity(content.len());
-    for line in content.lines() {
+    let mut lines = content.lines();
+    let last_line = lines.next_back();
+    let mut handle_line = |line: &str, newline: &str| {
         if let Some(caps) = BORING_LINES_REGEX.captures(line) {
             if &caps[2] == "#" {
                 result += &caps[1];
                 result += &caps[2];
                 result += &caps[3];
                 result += "\n";
```

Comment:
Is there a reason this newline isn't treated like the others?

Reply:
Oh, sorry about that — there's no reason to treat it differently.